salt/kubernetes-common/openstack-config.jinja

```diff
@@ -0,0 +1,16 @@
+[Global]
+auth-url={{ pillar['cloud']['config']['auth_url'] }}
+username={{ pillar['cloud']['config']['username'] }}
+password={{ pillar['cloud']['config']['password'] }}
+tenant-name={{ pillar['cloud']['config']['project'] }}
+domain-name={{ pillar['cloud']['config']['domain'] }}
+region={{ pillar['cloud']['config']['region'] }}
+[LoadBalancer]
+lb-version=v2
+subnet-id={{ pillar['cloud']['config']['subnet'] }}
```

Comment:
I am still not convinced that the sec groups stuff is safe - I need to look back at the outstanding bugs. Lets add that at a later date.